roles/balloonprinter/tasks/main.yml

```diff
@@ -0,0 +1,64 @@
+# windows updates
+#
+#
+#
+
+
+# set autologin
```

Comment:
```suggestion
```
Its already in the name of the task, IMO having this info in the `name` is always better so you also see it during the play and know whats happening without having to read the book